conda-recipe/run_test.sh

```diff
@@ -0,0 +1 @@
+nosetests -v metadatastore --cover-package metadatastore --with-coverage
```

Comment:
No, the problem I am trying to avoid is when `python setup.py install` and `conda install` produce different results